### PR TITLE
2.5.3.1 release notes

### PR DESCRIPTION
**Pull request by polarweasel**
Direct link to the [preview build of the release notes](https://deploy-preview-7527--infallible-bardeen-164bc9.netlify.app/latest/releases/whats-new/latest-releases/#v2-5-3-mar-16-2021)

### CODE REVIEW

**Comment by chirag-yb** [comment]
Features tagged incorrectly under Platform 
 [7126] PITR: Introduce snapshot schedule
 [7121] Extend yb-admin restore_snapshot to use a custom time
 
 Please remove these items
 [7342] [Platform] [UI] add Beta tag to HA tab + address feedback
[5383] Fix conflict resolution while adding read intents
[7372] Platform: Skip running periodic schedules when in follower mode
[7423] Platform: Fix AvailabilityZone::getByCode
[7488] Platform: Fix a sync backup bug due to File::renameTo behavior
7494] [YW] Hide Pause Universe in universe actions dropdown

Add two new items
Platform High Availability in beta status
https://github.com/yugabyte/yugabyte-db/issues/7475

File: docs/content/latest/releases/whats-new/latest-releases.md [hunk 1]
```diff
@@ -16,11 +16,110 @@ isTocNested: true
...
+
+```sh
+docker pull yugabytedb/yugabyte:2.5.3.1-bxx
+```
+
+### New Features
+
+#### Yugabyte Platform
+
+* [[7126](https://github.com/yugabyte/yugabyte-db/issues/7126)] PITR: Introduce snapshot schedule
```
Comment (bmatican) [inline docs/content/latest/releases/whats-new/latest-releases.md:54]: this is core db work, it's a building block for PITR (point in time restore)

also, not sure how to group this (new feature vs improvement), technically, PITR will be a net new feature, but right now it's a WIP / covers a small subset of usage

File: docs/content/latest/releases/whats-new/latest-releases.md [hunk 2]
```diff
@@ -16,11 +16,110 @@ isTocNested: true
...
+
+### Improvements
+
+#### Yugabyte Platform
+
+* [[5628](https://github.com/yugabyte/yugabyte-db/issues/5628)] [yugaware] [azure] Support regions with no availability zones
+* [[5807](https://github.com/yugabyte/yugabyte-db/issues/5807)] [Platform] Added toggle to show deleted backups
+* [[6373](https://github.com/yugabyte/yugabyte-db/issues/6373)] Platform: Pause/Resume universe. (#6846)
+* [[6962](https://github.com/yugabyte/yugabyte-db/issues/6962)] Platform: Exposes endpoint for downloading log files.
+* [[7121](https://github.com/yugabyte/yugabyte-db/issues/7121)] Extend yb-admin restore_snapshot to use a custom time
```
Comment (bmatican) [inline docs/content/latest/releases/whats-new/latest-releases.md:70]: also a core db item, this is the current CLI to trigger PITR

File: docs/content/latest/releases/whats-new/latest-releases.md [hunk 3]
```diff
@@ -16,11 +16,110 @@ isTocNested: true
...
+
+* [[7366](https://github.com/yugabyte/yugabyte-db/issues/7366)] YSQL: Allow getting current SQL query in pggate for debug logging
+* [[7418](https://github.com/yugabyte/yugabyte-db/issues/7418)] [[7463](https://github.com/yugabyte/yugabyte-db/issues/7463)] YSQL: Import Introduce the 'force' option for the Drop Database command.
+
+### Bug Fixes
+
+#### Yugabyte Platform
+
+* [[4742](https://github.com/yugabyte/yugabyte-db/issues/4742)] [Platform] Plotly tooltip is overlaping with no data on zero pozitions (on x or y)
+* [[5383](https://github.com/yugabyte/yugabyte-db/issues/5383)] Fix conflict resolution while adding read intents
```
Comment (bmatican) [inline docs/content/latest/releases/whats-new/latest-releases.md:82]: this is a core db bugfix

Comment (polarweasel) [reply docs/content/latest/releases/whats-new/latest-releases.md]: Removed at Chirag's request

**Comment by VijiYB** [comment]
Alex - Can you add this https://github.com/yugabyte/yugabyte-db/issues/7493 as known UI issue.  When there is no upgrade, a 0 appended to "Upgrade Software" under Actions menu.
@chirag-yb @streddy-yb

**Pull request merged**